### PR TITLE
Feature/12 seed data

### PR DESCRIPTION
**Pull request by TomasNiemczyk98**
## Summary

Added seeds for interview questions, changed documentation to mention how to update database

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactoring (non-breaking change to code and structure)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by PlesnikJakub** [comment]
Please explore the option of how to exclude seeds from migration. It would be useful for Prod.

**Comment by PlesnikJakub** [comment]
I would expect seed to be also executed manually if needed. There will be a need for adding more data to the seed so having it in migration would complicate things. 🤔

File: src/WebApi/Infrastructure/Migrations/20220202082648_InitalCreate.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 namespace Infrastructure.Migrations
 {
-    public partial class InitialCreate : Migration
+    public partial class InitalCreate : Migration
```
Comment (PlesnikJakub) [inline src/WebApi/Infrastructure/Migrations/20220202082648_InitalCreate.cs:7]: Should this be rather `InitialCreate` ? 🤔

**Review by PlesnikJakub — CHANGES_REQUESTED** [review]
I think there is only one typo and overall it looks good 😊

**Comment by PlesnikJakub** [comment]
Ignore sonar cloud duplicity warning, its not relevant in this case.

File: src/WebApi/Infrastructure/Migrations/20220202082648_InitalCreate.cs (hunk 1 above)
Comment (TomasNiemczyk98) [reply src/WebApi/Infrastructure/Migrations/20220202082648_InitalCreate.cs]: Fixed

**Review by PlesnikJakub — APPROVED** [review]

**Comment by PlesnikJakub** [comment]
I will merge this one due to permission to merge if the check fails.

**Pull request merged**